docs/misc/changelog.rst

```diff
@@ -21,6 +21,7 @@ New Features:
   when ``psutil`` is available
 - Saving models now automatically creates the necessary folders and raises appropriate warnings (@PartiallyTyped)
 - Refactored opening paths for saving and loading to use strings, pathlib or io.BufferedIOBase (@PartiallyTyped)
+- Introduced ``BaseModel`` abstract parent for ``BasePolicy``, which critics inherit from.
```

Comment:
isn't that a breaking change?

Reply:
I might be missing something, but I don't think this should break any user code: `BasePolicy` still has the same interface as before, you can just choose to use a subset of it via `BaseModel`. I guess `ContinuousCritic` now has a slightly different interface (e.g. no `predict()`, but it would have just raised a `NotImplementedError` before).

Reply:
fair enough ;)